src/compilation/build.sh

```diff
@@ -208,6 +211,64 @@ function build_ncurses() {
     popd > /dev/null
 }
 
+function build_python() {
+    # Build python.
+    #
+    # Parameters:
+    # $1: python package directory
+    # $2: target architecture
+    #
+    # Echoes:
+    # The python build directory
+    #
+    # Returns:
+    # 0: success
+    # 1: failure
+    local python_dir="$1"
+    local target_arch="$2"
+    local python_lib_dir="$(realpath "$python_dir/build-$target_arch")"
+
+    echo "$python_lib_dir"
```

Comment:
do you not want to redirect to stderr as well here?

Reply:
No, this is how you "return" values that aren't 0-255 from bash functions.
You capture the regular output on call.